.gitignore

```diff
@@ -11,3 +11,5 @@
 /test/out
 
 /test/work
+
+/metasv/run_after_age.py
```

Comment:
You should not include that file in the package if it is just for your testing purposes. It should also be missing from gitignore since it is not being generated automatically. You can just avoid committing this file to the repo which will be the default since the file has not been added to the repo.